<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_FontLoad</class>
 <widget class="QDialog" name="Dialog_FontLoad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>563</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>548</width>
    <height>563</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>548</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog_FontLoad</string>
  </property>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>521</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_FontFamily">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Font family</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Text size</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Size">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>10</y>
     <width>71</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_Style">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>241</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Style</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Metrix">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>60</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>181</width>
     <height>0</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Height">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>200</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>32</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>New height</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>71</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>New width</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_Width">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>61</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimum">
    <number>5</number>
   </property>
   <property name="maximum">
    <number>32</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Data items lenght</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_DigitLenght">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>200</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_StringDataOrientation">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>86</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>151</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Data ordering in a string</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_FileView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>521</width>
     <height>91</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Test">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>190</y>
     <width>84</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Test</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Test">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>230</y>
     <width>71</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_Separate">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>380</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Separate symbols</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_FileName">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>380</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Display">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>521</width>
     <height>130</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>521</width>
     <height>130</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
